src/app/pages/videos/videos-list/videos-list-container/videos-list-container.component.html

```diff
@@ -0,0 +1,5 @@
+<div scrollTracker (scrollingFinished)="onScrollingFinished()">
+<ng-container *ngFor="let videos of videosList$|push" >
+    <app-video-card [video]="videos"></app-video-card>
+</ng-container>
+</div>
```

Comment:
We can move its contents to videos component. Otherwise the videos component is also useless.

Let me know your thoughts

Reply:
Sure i will do the same..will move the contents